site/components/footer/index.js

```diff
@@ -20,31 +21,43 @@ import mapSVG from './SVG/map.svg';
 
 const cx = classnames.bind(styles);
 
+const trackAnalytics = title => () => ReactGA.event({
+  category: 'FooterNavigation',
+  action: title,
+  label: 'click',
+});
```

Comment:
Use the `navigating` prop https://github.com/redbadger/website-honestly/blob/f7b21ea17c8f21cdccb982a08896922d0378459e/site/components/header/small-screen-nav.js#L83 instead of span onclick
This handles all those things like right clicking context menu